spread.yaml

```diff
@@ -522,4 +522,44 @@ suites:
             distro_purge_package qemu genisoimage sshpass
             snap remove ubuntu-image
 
+    tests/performance/:
+        summary: Performance tests for snapd
+        manual: true
+        environment:
+            RES_FILE: "$PROJECT_PATH/test.perf"
+        prepare: |
+            . "$TESTSLIB/prepare.sh"
+            if [[ "$SPREAD_SYSTEM" == ubuntu-core-16-* ]]; then
+                prepare_all_snap
+            else
+                prepare_classic
+            fi
+
+            . "$TESTSLIB/perf.sh"
+            create_results_file $RES_FILE
+
+        prepare-each: |
+            "$TESTSLIB/reset.sh" --reuse-core
+            . $TESTSLIB/prepare.sh
+            if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
+                prepare_each_classic
+            fi
+
+        restore: |
+            "$TESTSLIB/reset.sh" --store
+            if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
+                . $TESTSLIB/pkgdb.sh
+                distro_purge_package snapd
+                if [[ "$SPREAD_SYSTEM" != opensuse-* ]]; then
```

Comment:
We have a similar check in `prepare-restore.sh`, you need to make it:
```
        if [[ "$SPREAD_SYSTEM" != opensuse-* && "$SPREAD_SYSTEM" != arch-* ]]; then
            # A snap-confine package never existed on openSUSE or Arch
            distro_purge_package snap-confine
        fi
```

Edit: this is already included in `--restore-suite`